src/Integration.Vsix/Analysis/AnalysisStatusNotifier.cs

```diff
@@ -67,11 +69,16 @@ public void AnalysisFailed(Exception ex)
 
         public void AnalysisFailed(string failureMessage)
         {
-            logger.WriteLine(AnalysisStrings.MSG_AnalysisFailed, filePath, failureMessage);
+            Log(AnalysisStrings.MSG_AnalysisFailed, filePath, failureMessage);
 
             Notify(AnalysisStrings.Notifier_AnalysisFailed, false);
         }
 
+        private void Log(string messageFormat, params object[] args)
+        {
+            logger.WriteLine($"[{analyzerName}]: " + messageFormat, args);
+        }
+
         private void Notify(string messageFormat, bool showSpinner)
```

Comment:
StatusBar remains unchanged as it needs to be a short message. Only the output window is changed.

![image](https://user-images.githubusercontent.com/60586879/204024844-2b5bd323-5bf8-4a08-a95c-f51fc79d8826.png)